plugins/inputs/squid/dev/docker-compose.yml

```diff
@@ -0,0 +1,21 @@
+version: '3'
+
+services:
+  squid:
+    image: b4tman/squid:latest
+    restart: always
+    volumes:
+      - ./squid.conf:/etc/squid/squid.conf:ro
+    ports:
+      - "3128:3128"
+  telegraf:
+    image: glinton/scratch
+    depends_on:
+      - squid
+    volumes:
+      - ./telegraf.conf:/telegraf.conf:ro
+      - ../../../../telegraf:/telegraf:ro
```

Comment:
does this need execute permissions to be able to run?

Reply:
Not sure I follow. This works just fine, and I think preventing containers from modifying files on the host is considered a better practice.